docs/registry.md

```diff
@@ -0,0 +1,118 @@
+ # Registry Support for SOCI
+
+SOCI is supported by most registries. The notable exception to this is Duckerhub, which currently does not support subject fields in image manifests.
```

Comment:
Can we rephrase all the various uses of "registry supports soci" to refer more to the features we depend on and less to us specifically? This phrasing suggests that we want registries to do something special to support us in particular. I want to emphasize that we rely on interoperable standards. Perhaps something like "The features that SOCI relies on are available in most registries" here, but less verbose rephrasing in other instances further down the doc. Perhaps "is compatible with" rather than "supports" or "is supported by".

Reply:
That's a good call out. I'm going to change "support" to "compatible with" when talking about SOCI and only use the term "support" in reference the specific OCI spec features that we rely on.

Reply:
Can you take another look at let me know if you like the new phrasing?

Reply:
Looks great. Love the phrasing of it as registries supporting features rather than us supporting registries.